src/keymap-manager.coffee

```diff
@@ -513,6 +519,15 @@ class KeymapManager
     dispatchedExactMatch = null
     partialMatches = @findPartialMatches(partialMatchCandidates, target)
 
+    if @pendingPartialMatchedModifierKeystrokes? and isModifierKeyup(keystroke)
+      for binding in @pendingPartialMatchedModifierKeystrokes
+        binding_mod_keyups = getModKeys(binding.keystrokeArray[binding.keystrokeArray.length-1])
```

Comment:
Could this be encapsulated in a method in the binding that takes the keystroke string?